functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
@sanderstad and I spoke via Slack earlier. He'll be modifying the other `*-DbaAgentSchedule` commands to allow creation of a schedule without an associated job.

I've also addressed @wsmelton 's comment. Given that, can this be merged?